Source/WebCore/editing/Editor.cpp

```diff
@@ -2071,6 +2072,81 @@ void Editor::toggleUnderline()
     command("ToggleUnderline"_s).execute();
 }
 
+void Editor::setTextAlignmentForChangedBaseWritingDirection(WritingDirection direction)
+{
+    // Note that the passed-in argument is the direction that has been changed to by
+    // some code or user interaction outside the scope of this function. The former
+    // direction is not known, nor is it required for the kind of text alignment
+    // changes done by this function.
+    //
+    // Rules:
+    // When text has no explicit alignment, set to alignment to match the writing direction.
```

Comment:
did you mean `set alignment to match` here?
```suggestion
    // When text has no explicit alignment, set **to alignment to match** the writing direction.
```

Reply:
Oh, good catch — that seems like a typo (I just moved this comment from `EditorIOS.mm` as-is).

Thanks for the review!